config/config.go

```diff
@@ -88,6 +96,9 @@ type Config struct {
 	// convenience fields to access the source subnet and chain IDs after initialization
 	sourceSubnetIDs     []ids.ID
 	sourceBlockchainIDs []ids.ID
+
+	// convenience field to store the Warp quorum figures for each subnet
+	warpQuorum map[ids.ID]WarpQuorum
```

Comment:
I think we only need the quorum values for the destination chains right? If so, I'd recommend putting this within `DestinationSubnet` rather than an extra map in `Config`

Reply:
Made this change as suggested. One thing to note is that `Config.GetWarpQuorum` now iterates over the list of destination subnets rather than indexing into a map. I thought about adding a `map[ids.ID]DestinationSubnet` to `Config`, but since the length of that list is effectively constant, iterating is fine.